Gemfile.lock

```diff
@@ -89,35 +98,45 @@ GEM
       money (~> 6.7)
       nokogiri (~> 1.5)
       sanitize (~> 2.1.0)
-    govuk-content-schema-test-helpers (1.6.0)
-      json-schema (~> 2.8.0)
-    govuk-lint (3.4.0)
+    govuk-content-schema-test-helpers (1.5.0)
+      json-schema (~> 2.5.1)
+    govuk-lint (3.6.0)
       rubocop (~> 0.51.0)
       rubocop-rspec (~> 1.19.0)
       scss_lint
-    govuk_frontend_toolkit (7.2.0)
+    govuk_ab_testing (2.4.1)
+    govuk_frontend_toolkit (7.3.0)
       railties (>= 3.1.0)
       sass (>= 3.2.0)
+    govuk_navigation_helpers (9.0.0)
```

Comment:
How much code duplication/headache would we have by not making this a dependency?

Reply:
We'd need to copy over these two files:
https://github.com/alphagov/govuk_navigation_helpers/blob/master/lib/govuk_navigation_helpers/content_item.rb

https://github.com/alphagov/govuk_navigation_helpers/blob/master/lib/govuk_navigation_helpers/grouped_related_links.rb